infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/event/PodEvent.java

```diff
@@ -11,22 +11,23 @@
 package org.eclipse.che.workspace.infrastructure.kubernetes.namespace.event;
 
 import java.util.Objects;
+import org.eclipse.che.commons.annotation.Nullable;
 
 /**
  * The event that should be published when container change occurs, e.g. image pulled, container
  * started.
  *
  * @author Sergii Leshchenko
  */
-public class ContainerEvent {
+public class PodEvent {
   private final String podName;
-  private final String containerName;
+  @Nullable private final String containerName;
```

Comment:
It would be even more useful to have `@Nullable` annotation on `#getContainerName`. Also please note in javadoc of `#getContainerName` method that it may be nullable and when

Reply:
fixed